src/Controls/src/Core/Layout.cs

```diff
@@ -307,7 +307,10 @@ internal static void LayoutChildIntoBoundingRegion(View child, Rectangle region,
 			region.Y += margin.Top;
 			region.Height -= margin.VerticalThickness;
 
-			child.Layout(region);
+			if (child is IFrameworkElement fe)
```

Comment:
Arrange calls Layout and does extra stuff to make it work if it has purely handler based controls. We might not need this if statement